repository/l.json

```diff
@@ -32,6 +32,19 @@
 				}
 			]
 		},
+		{
+			"name": "Language - Latin",
+			"description": "Latin spelling dictionary for Sublime Text",
+			"author": ["fawzisal"],
```

Comment:
You don't need to specify the "author" since "details" already points to your GitHub repo. Also the "description" key is better left out, we also take that from the repo.